reframe/core/schedulers/slurm.py

```diff
@@ -31,6 +31,9 @@ class SlurmJobState(sched.JobState):
 SLURM_JOB_SUSPENDED   = SlurmJobState('SUSPENDED')
 SLURM_JOB_TIMEOUT     = SlurmJobState('TIMEOUT')
 
+# Number of _update_state calls per which _cancel_if_blocked is called
```

Comment:
I don't like this comment very much; it contradicts with the name of the variable. If we use the `SACCT_SQUEUE_RATIO` as a name, I'd like to see a comment describing the rationale behind this instead, because what this variable controls is obvious from its name.